CHANGELOG.md

```diff
@@ -1,9 +1,27 @@
 ## master
 
-* Relay will now throw an invariant if range behavior keys are unsorted.
+## 0.6.0 (December 4, 2015)
+
+* Bump the `babel-relay-plugin` version to v0.5.0 (now Babel 6 compatible).
 * The keys in `rangeBehaviors` are now compared against the *sorted* filter
   arguments of a field. For the field `foo(first: 10, b:true, a:false)` the
   matching range behavior key will be `'a(false).b(true)'`.
+* Relay will now throw an invariant if range behavior keys are unsorted.
+* Fragments are now supported in mutation fat queries.
+* Value substitutions via expression interpolation in `Relay.QL` queries no
```

Comment:
No one has asked for this in OSS that I'm aware of, and it's generally an anti-pattern. It (should?) require a special option in babel-relay-plugin to enable and I don't know if we want to to advertise it.